usr/sbin/p11sak/p11sak.c

```diff
@@ -1294,11 +1502,15 @@ static CK_RV pub_key_print_attributes(CK_SESSION_HANDLE session,
     CK_BBOOL a_wrap;
     CK_ULONG bs = sizeof(CK_BBOOL);
 
-    CK_ATTRIBUTE bool_tmplt[] = { { CKA_TOKEN, &a_token, bs }, { CKA_PRIVATE,
-            &a_private, bs }, { CKA_MODIFIABLE, &a_modifiable, bs }, {
-    CKA_DERIVE, &a_derive, bs }, { CKA_LOCAL, &a_local, bs }, {
-    CKA_ENCRYPT, &a_encrypt, bs }, { CKA_VERIFY, &a_verify, bs }, {
-    CKA_WRAP, &a_wrap, bs } };
+    CK_ATTRIBUTE bool_tmplt[] = { 
```

Comment:
same comment about the use of sizeof() as above.

Reply:
Done